daemon/restart.go

```diff
@@ -14,7 +14,7 @@ import (
 // timeout, ContainerRestart will wait forever until a graceful
 // stop. Returns an error if the container cannot be found, or if
 // there is an underlying error at any stage of the restart.
-func (daemon *Daemon) ContainerRestart(name string, seconds *int) error {
+func (daemon *Daemon) ContainerRestart(name string, seconds *int, checkpoint string, checkpointDir string) error {
```

Comment:
This could probably use an `options` type as well

Reply:
It would appear to me that it should be kept consistent with [ContainerStart](https://github.com/moby/moby/blob/master/daemon/start.go#L18).